sensor_msgs/PointCloud2 input
---
bool result
